### PR TITLE
Created templates for Terrarium

### PR DESCRIPTION
**Pull request by mserykh**

### CODE REVIEW

**Comment by mserykh** [comment]
Issue #62

File: .github/ISSUE_TEMPLATE/3.1.1.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+## Pre-Lecture Quiz
+
+*A warm-up quiz about HTML*
+
+Complete this quiz in class
+
+1. HTML stands for 'HyperText Mockup Language'
+
+- [ ] [true]
```
Comment (dev-experience) [inline .github/ISSUE_TEMPLATE/3.1.1.md:21]: Square brackets around answers and questions are unnecessary in my opinion.
@mserykh, could you please remove them in all pre- and post-lecture quizes?

> I know that you didn't put them there (they were in original lessons).

**Review by dev-experience — CHANGES_REQUESTED** [review]
Please check my comment.

**Comment by dev-experience** [comment]
@mserykh I removed it from the project board because you have the parent issue #62 there. Having them both together on the board is confusing.

File: .github/ISSUE_TEMPLATE/3.1.1.md (hunk 1 above)
Comment (mserykh) [reply .github/ISSUE_TEMPLATE/3.1.1.md]: @dev-experience done

**Review by dev-experience — APPROVED** [review]
@mserykh Great! Thanks a lot 👍🏽

**Pull request merged**